### ISSUE TITLE
About SRE models

### ISSUE THREAD

**Issue by PES2g** [mycrazycracy/tf-kaldi-speaker#2]
How could i request the pretrained SRE models ?

**Comment by mycrazycracy** [comment]
Hi, @PES2g I'm too busy to publish the SRE models these days. Please use the VoxCeleb models first. I will publish the SRE models once I got time.

**Comment by dev0x13** [comment]
Hello! Are there any updates on how to receive pretrained SRE models? Thank you!

**Comment by mycrazycracy** [comment]
Yes, I will upload the pretrained SRE models today. The models are trained using large margin softmax. Hope that helps!

**Comment by dev0x13** [comment]
Thank you very much!

**Comment by PES2g** [comment]
Thanks.
